unstract/connectors/src/unstract/connectors/filesystems/zs_dropbox/zs_dropbox.py

```diff
@@ -74,3 +75,19 @@ def test_credentials(self) -> bool:
             logger.error(f"Test creds failed: {e}")
             raise ConnectorError(str(e))
         return True
+
+    @staticmethod
+    def get_connector_root_dir(input_dir: str, **kwargs: Any) -> str:
+        """Get roor dir of zs dropbox."""
+        return f"/{input_dir.strip('/')}"
+
+    def create_dir_if_not_exists(self, input_dir: str) -> None:
+        """Create roor dir of zs dropbox if not exists."""
+        fs_fsspec = self.get_fsspec_fs()
+        try:
+            fs_fsspec.isdir(input_dir)
```

Comment:
[SUGGESTION] I belive fsspec has method to check and create a directory - `auto_mkdir`
We can leverage it, if possible. [Ref](https://filesystem-spec.readthedocs.io/en/latest/api.html)

Reply:
@harini-venkataraman have handled that on connector class